jsk_arc2017_common/node_scripts/publish_shelf_bin_bbox.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python
```

Comment:
But there is no name information in the published message, isn't it? because it is jsk_recognition_msgs/BoundingBoxArray. If so why you can't use the bounding_box_array_publisher.py?